doc/Doxyfile

```diff
@@ -1486,7 +1486,7 @@ EXT_LINKS_IN_WINDOW    = NO
 
 FORMULA_FONTSIZE       = 10
 
-# Use the FORMULA_TRANPARENT tag to determine whether or not the images
+# Use the FORMULA_TRANSPARENT tag to determine whether or not the images
```

Comment:
I think this can just be sent upstream as well. It'll be pulled in if/when we regenerate the Doxyfile next.